app/javascript/app/document-capture/hooks/use-assets.js

```diff
@@ -0,0 +1,14 @@
+import { useContext } from 'react';
+import AssetContext from '../context/assets';
+
+function useImage() {
+  const strings = useContext(AssetContext);
+  const imageStrings = strings.images || {};
+  const imageTag = (key) => {
+    const resolvedImage = imageStrings[key];
+    return resolvedImage !== undefined ? resolvedImage : key;
+  };
+  return imageTag;
+}
+
+export { useImage };
```

Comment:
In separate work I came across the need to recreate an existing shared view element in React (the accordion), and it occurred to me that it would be a nice abstraction to be able to use an image as a regular `<img>` HTML tag, in a way that the application would substitute the digest-modified path automatically. I was considering this a new `Image` component.

Example usage:

```jsx
<Image
  src="plus.svg"
  alt={ t('image_description.accordian_plus_buttom') }
  width={ 16 }
  className="plus-icon display-none"
/>
```

Recreating this:

https://github.com/18F/identity-idp/blob/982dc4af12db8d353894c7cb6eaf8ed2395e2da5/app/views/shared/_accordion.html.slim#L17-L18

Repurposing `src` as a reference to an asset path might be a _bit_ too magical, so naming it something like `assetPath` could be more appropriate.

As far as it relates here, I still think we'll need the asset context, but I'm not yet sure about these hooks. It seems like if we had an `Image` component abstraction, it wouldn't be too much a regular bother to reference the context using `useContext`, without any intermediary `useAsset` or `useImage` hooks.